src/Microsoft.Fx.Portability.MetadataReader/Microsoft.Fx.Portability.MetadataReader.project.json

```diff
@@ -1,18 +1,18 @@
 ﻿{
   "supports": {
-    "net46.app": { },
-    "dnxcore50.app": { }
+    "net46.app": { }
   },
   "dependencies": {
     "MicroBuild.Core": "0.2.0",
     "Microsoft.NETCore": "5.0.0",
-    "Microsoft.NETCore.Portable.Compatibility": "1.0.1-beta-23516",
-    "System.Reflection.Metadata": "1.2.0-rc2-23713",
-    "System.Security.Cryptography.Algorithms": "4.0.0-beta-23328"
+    "Microsoft.NETCore.Portable.Compatibility": "1.0.0",
```

Comment:
I originally had an issue with it, and found the 1.0.0 works, so figured go with the lowest necessary version. I have another branch I've got things working with DotNet CLI, so this will be replaced soon, anyway